### PR TITLE
fix missing arg to logger

### PR DESCRIPTION
**Pull request by sjamgade**

### CODE REVIEW

File: astacus/coordinator/plugins/base.py [hunk 1]
```diff
@@ -445,7 +445,7 @@ async def run_step(self, cluster: Cluster, context: StepsContext) -> None:
             # Since deltas can be uploaded from a different set of nodes than the base backup,
             # explicitly re-match current nodes to delta manifest nodes on each restore.
             if self.partial_restore_nodes and not self.contains_partial_restore_hostnames(delta_manifest):
-                logger.info("Skipped %s, because no data from it maps to partial restore nodes")
+                logger.info("Skipped %s, because no data from it maps to partial restore nodes", delta_manifest)
```
Comment (kmichel-aiven) [inline astacus/coordinator/plugins/base.py:448]: We probably should just print `delta_name` instead of the entire `delta_manifest` ?

**Review by kmichel-aiven — CHANGES_REQUESTED** [review]

File: astacus/coordinator/plugins/base.py (hunk 1 above)
Comment (sjamgade) [reply astacus/coordinator/plugins/base.py:448]: yes, the manifest object seems pretty big, but delta_name does not tell much about the manifest, or does it ?

Comment (kmichel-aiven) [reply astacus/coordinator/plugins/base.py:448]: It does not tell much, if you want to be the most helpful you probably need to extract all the data used by `contains_partial_restore_hostnames`: `[pr.backup_hostname for pr in self.partial_restore_nodes]` and `[sr.hostname in partial_restore_hostnames for sr in manifest.snapshot_results]`.

Comment (sjamgade) [reply astacus/coordinator/plugins/base.py]: went with `delta_manifest.start` it tells something more individually identifiable

**Review by kmichel-aiven — APPROVED** [review]

**Pull request merged**